helm/korifi/gateway-api-router/deployment.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: korifi-gateway-api-router-manager
+  labels:
+    app: korifi-gateway-api-router
+  namespace: { { .Release.Namespace } }
+spec:
+  replicas: { { .Values.controllers.replicas | default 1 } }
+  selector:
+    matchLabels:
+      app: korifi-gateway-api-router
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+        prometheus.io/path: /metrics
+        prometheus.io/port: "8080"
+        prometheus.io/scrape: "true"
+        checksum/config: { { tpl ($.Files.Get "gateway-api-router/configmap.yaml") $ | sha256sum } }
+      labels:
+        app: korifi-gateway-api-router
+    spec:
+      containers:
+        - name: manager
+          env:
+            - name: ROUTECONTROLCONFIG
+              value: /etc/korifi-gateway-api-router-config
+          image: { { .Values.gateway-api-router.image } }
+          imagePullPolicy: IfNotPresent
+          { { - if .Values.debug } }
+          command:
+            - "/dlv"
+          args:
+            - "--listen=:40000"
+            - "--headless=true"
+            - "--api-version=2"
+            - "exec"
+            - "/manager"
+            - "--continue"
+            - "--accept-multiclient"
+            - "--"
+            - "--health-probe-bind-address=:8081"
+            - "--leader-elect"
+          { { - else } }
+          args:
+            - "--health-probe-bind-address=:8081"
+            - "--leader-elect"
+          { { - end } }
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          ports:
+            - containerPort: 9443
+              name: webhook-server
```

Comment:
After we move webhooks back to the core controllers we will no longer need a webhook server